src/operator/effectivepolicy/groupreconciler.go

```diff
@@ -73,10 +73,16 @@ func (g *GroupReconciler) getAllServiceEffectivePolicies(ctx context.Context) ([
 	// Extract all services from intents
 	services := goset.NewSet[serviceidentity.ServiceIdentity]()
 	for _, clientIntent := range intentsList.Items {
+		if !clientIntent.DeletionTimestamp.IsZero() {
+			continue
+		}
 		service := serviceidentity.ServiceIdentity{Name: clientIntent.Spec.Service.Name, Namespace: clientIntent.Namespace}
 		services.Add(service)
 		serviceToIntent[service] = clientIntent
 		for _, intentCall := range clientIntent.GetCallsList() {
+			if intentCall.IsTargetServerKubernetesService() {
```

Comment:
This would mean that the service is not counted as a server, right?
So how are ingress policies created for it?

Reply:
They are created using the same method as the egress ones. This is something we can change when we start using existing pod selectors instead of labeling them ourselves.